docs/UserGuide.adoc

```diff
@@ -57,25 +57,42 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Format: `add n/NAME [p/PHONE_NUMBER]... [e/EMAIL]... [a/ADDRESS]... [t/TAG]...`
 
 [TIP]
-A person can have any number of tags (including 0)
+A person can have any number of tags,phone numbers, emails or addresses (including 0)
 
 Examples:
 
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Bob p/91294232`
 
 === Listing all persons : `list`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Shows a list of persons in the address book. +
+Format: `list [m/MODE] [tag/TAG]`
+
+****
+* MODE can be either "recent", "frequent" or "tag". If MODE is set to "tag", the field [tag/TAG] must also be included.
+* If the [m/MODE] field is empty, a list of all persons is shown
+* Using MODE = "recent" will return a list of the 5 most recently accessed entries
+* Using MODE = "frequent" will return a list of the 5 most frequently accessed entries
+* Using MODE = "tag" will return a list of all person that are tagged with TAG
+****
+
+Examples:
+
+* `list`
```

Comment:
Desc of this command (lists all persons)?

Reply:
Added this